docs/content/integrations/dbt/reference.mdx

```diff
@@ -532,6 +532,39 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Emit dbt table schema as materialization metadata <Experimental />
+
+Dagster allows you to emit table schema [materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata), which includes the column names and data types of your materialized dbt models, seeds, and snapshots.
+
+With this metadata, you can view documentation for all columns and not just columns described in your dbt project.
+
+To collect this metadata for your dbt project, specify the `dagster` dbt package in your `dependencies.yml`, and then run `dbt deps`.
```

Comment:
Does this mean that they'll also need to run dbt dips in CI?

Reply:
Yeah. Running any command (e.g. `dbt build`, `dbt compile`) will fail unless `dbt deps` is run. We already recommend running `dbt deps`: https://docs.dagster.io/integrations/dbt/reference#deploying-a-dagster-project-with-a-dbt-project.